### PR TITLE
[ja] Translate concepts/scheduling-eviction/scheduling-framework/ into Japanese

### LINKED ISSUE #31886: [ja] Translate concepts/scheduling-eviction/scheduling-framework/ into Japanese
**This is a Feature Request**

**What would you like to be added**

Translate [concepts/scheduling-eviction/scheduling-framework/](https://kubernetes.io/docs/concepts/scheduling-eviction/scheduling-framework/) into Japanese.

**Why is this needed**
This page has not been translated.

**Comments**

/language ja
/assign

### PR DESCRIPTION
**Pull request by 196Ikuchil**
Resolve #31886

### CODE REVIEW

**Comment by 196Ikuchil** [comment]
If I should translate "Scheduling Cycle" and "Binding Cycle" as 「スケジューリングサイクル」「バインディングサイクル」, please let me know. I will replace it and recommit.

File: content/ja/docs/concepts/scheduling-eviction/scheduling-framework.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+と`Unreserve`と呼ばれる2つの情報スケジューリングフェーズを返します。
+実行状態を保持するプラグイン（別名「ステートフルプラグイン」）は、これらのフェーズを使用して、Podに対してNodeのリソースが予約されたり予約解除された場合に、スケジューラーから通知を受け取ります。
+
+Reserveフェーズは、スケジューラーが実際にPodを指定されたNodeにバインドする前に発生します。このフェーズはスケジューラーがバインドが成功するのを待つ間にレースコンディションの発生を防ぐためにあります。
+各Reserveプラグインの`Reserve`メソッドは成功することも失敗することもあります。もしどこかの`Reserve`メソッドの呼び出しが失敗すると、後続のプラグインは実行されず、Reserveフェーズは失敗したものとみなされます。全てのプラグインの`Reserve`メソッドが成功した場合、Reserveフェーズは成功とみなされ、残りのScheduling CycleとBinding Cycleが実行されます。
+
+Unreserveフェーズは、Reserveフェーズまたは後続のフェーズが失敗した場合に、呼び出されます。この時、**全ての**Reserveプラグインの`Unreserve`メソッドが、`Reserve`メソッドの呼び出された逆の順序で実行されます。このフェーズは予約されたPodに関連する状態をクリーンアップするためにあります。
+
+{{< caution >}}
+`Unreserve`メソッドの実装は冪等性を持つべきであり、失敗してはなりません。
```
Comment (196Ikuchil) [inline content/ja/docs/concepts/scheduling-eviction/scheduling-framework.md:110]: This translation may not be good. Please review me.

File: content/ja/docs/concepts/scheduling-eviction/scheduling-framework.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+PreFilterプラグインは、Podに関する情報を前処理したり、クラスターやPodが満たすべき特定の条件をチェックするために使用されます。PreFilterプラグインがエラーを返した場合、Scheduling Cycleは中断されます。
+
+### Filter
+
+FilterプラグインはPodを実行できないNodeを候補から除外します。各Nodeに対して、スケジューラーは設定された順番でFilterプラグインを呼び出します。いずれかのFilterプラグインが途中でそのNodeを実行不可能とした場合、残りのプラグインではそのNodeは呼び出されません。Nodeは同時に評価されることがあります。
+
+### PostFilter {#post-filter}
+
+PostFilterプラグインはFilterフェーズで、Podに対して実行可能なNodeが見つからなかった場合にのみ呼び出されます。このプラグインは設定された順番で呼び出されます。もしいずれかのPostFilterプラグインが、あるNodeを「スケジュール可能(Schedulable)」と目星をつけた場合、残りのプラグインは呼び出されません。典型的なPostFilterの実装はプリエンプション方式で、他のPodを先取りして、Podをスケジューリングできるようにしようとします。
```
Comment (196Ikuchil) [inline content/ja/docs/concepts/scheduling-eviction/scheduling-framework.md:55]: This translation may not be good. Please review me.

File: content/ja/docs/concepts/scheduling-eviction/scheduling-framework.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+content_type: concept
+weight: 90
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.19" state="stable" >}}
+
+スケジューリングフレームワークはKubernetesのスケジューラーに対してプラグイン可能なアーキテクチャです。
+このアーキテクチャは、既存のスケジューラーに新たに「プラグイン」としてAPI群を追加するもので、プラグインはスケジューラー内部にコンパイルされます。このAPI郡により、スケジューリングの「コア」の軽量かつ保守しやすい状態に保ちながら、ほとんどのスケジューリングの機能をプラグインとして実装することができます。このフレームワークの設計に関する技術的な情報についてはこちらの[スケジューリングフレームワークの設計提案][kep]をご覧ください。
```
Comment (kakts) [inline content/ja/docs/concepts/scheduling-eviction/scheduling-framework.md:12]:
```suggestion
このアーキテクチャは、既存のスケジューラーに新たに「プラグイン」としてAPI群を追加するもので、プラグインはスケジューラー内部にコンパイルされます。このAPI群により、スケジューリングの「コア」の軽量かつ保守しやすい状態に保ちながら、ほとんどのスケジューリングの機能をプラグインとして実装することができます。このフレームワークの設計に関する技術的な情報についてはこちらの[スケジューリングフレームワークの設計提案][kep]をご覧ください。
```

File: content/ja/docs/concepts/scheduling-eviction/scheduling-framework.md (hunk 1 above)
Comment (kakts) [reply content/ja/docs/concepts/scheduling-eviction/scheduling-framework.md:110]: Thank you for you translation.

>Caution: The implementation of the Unreserve method in Reserve plugins must be idempotent and may not fail.

may not は 〜してはいけない　という意味通りかと思いますが、表現が難しいですね
ちょっとニュアンス変わるかもですが、下記のような内容はどうでしょうか？
```suggestion
`Unreserve`メソッドの実装は冪等性を持つべきであり、この処理で問題があった場合に失敗させてはなりません。
```

**Comment by 196Ikuchil** [comment]
Hi @kakts, Thank you for your review!

**Comment by kakts** [comment]
@196Ikuchil Thank you for your contribution. 
/lgtm

**Comment by nasa9084** [comment]
sorry for late reviewing~
/approve

**Pull request merged**